src/app/routes/cpsAsset/getInitialData/index.js

```diff
@@ -46,7 +47,7 @@ const formatPageData = pipe(
   only([STORY_PAGE], insertPodcastPromo),
 );
 
-const processOptimoBlocks = toggles =>
+const processOptimoBlocks = toggles => () =>
```

Comment:
Don't you need to add service in the params?

Reply:
I've kept it empty for now as I've commented out the `addExperimentPlaceholderBlocks(service)` function. When working locally, it'll need uncommented and then `service` will need added in the params. I've kept it out just to avoid the `no unused vars` lint error.

Reply:
Yh disabling eslint on that line might be better in my opinion.